client-participation/js/strings/de_de.js

```diff
@@ -20,7 +20,7 @@ s.modImportant = "Wichtig";
 s.modSubmitInitialState = "Überspringen (nichts davon ist zutreffend), nächstes Statement";
 s.modSubmit = "Fertig, nächstes Statement";
 
-s.x_wrote = "schrieb:";
+s.x_wrote = "hat geschrieb:";
```

Comment:
hat geschrieben

Reply:
Obwohl hier vermutlich "schrieb" die gängigere Form ist.

Reply:
Danke für die Korrektur. Dumme Frage eine GuiHub Noobs: Wie ändere ich den Pull Request nun?

Zur Frage "hat geschrieben" vs. "schrieb": Ich verstehe den Einwand. Jeder Deutschlehrer würde wohl in der Schriftsprache "schrieb" bevorzugen. Ausschlaggebend war für mich am Ende, dass die Software möglichst inklusiv sein sollte. In den Regeln zur "leichten Sprache" wird vorgeschlagen, nur Präsens und Perfekt zu verwenden und niemals Präteritum. Es ist wohl für nicht-Muttersprachler leichter...warum auch immer....

Reply:
For linguistically-impoverished english folks, eager to follow along 🙂 

> > Obwohl hier vermutlich "schrieb" die gängigere Form ist.
>
>  Although "wrote" is probably the more common form here.

> > Danke für die Korrektur. Dumme Frage eine GuiHub Noobs: Wie ändere ich den Pull Request nun?
> >
> > Zur Frage "hat geschrieben" vs. "schrieb": Ich verstehe den Einwand. Jeder Deutschlehrer würde wohl in der Schriftsprache "schrieb" bevorzugen. Ausschlaggebend war für mich am Ende, dass die Software möglichst inklusiv sein sollte. In den Regeln zur "leichten Sprache" wird vorgeschlagen, nur Präsens und Perfekt zu verwenden und niemals Präteritum. Es ist wohl für nicht-Muttersprachler leichter...warum auch immer....
>
> Thanks for the correction. Stupid question from a GuiHub noob: How do I change the pull request now?
>
> Regarding the question "has written" vs. "wrote": I understand the objection. Every German teacher would probably prefer "wrote" in written language. In the end, the decisive factor for me was that the software should be as inclusive as possible. In the rules for "easy language" it is suggested to use only present and perfect tense and never past tense. It's probably easier for non-native speakers...for whatever reason....

Reply:
re: changing PR. the easiest way to make a "suggestion" (which anyone can do, including me), is to click the little "plus-minus" button in one of these line comments. This adds some text to the comment box:

<img width="979" alt="Screen Shot 2022-10-24 at 3 14 49 PM" src="https://user-images.githubusercontent.com/305339/197547406-d367a09e-a969-4940-b4b6-970bbedcb854.png">

    ```suggestion
    s.x_wrote = "hat geschrieb:";
    ```

If I replace the text with a change (like below), it renders like the box below, and can be merged in with a button-click:

    ```suggestion
    s.x_wrote = "schrieb:";
    ```

```suggestion
s.x_wrote = "schrieb:";
```